specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2022-10/schemaregistry.json

```diff
@@ -523,14 +508,18 @@
     },
     "SchemaVersions": {
       "type": "object",
-      "description": "Array received from the registry containing the list of versions for specific schema.",
+      "description": "Object received from the registry containing the list of schema versions and link to next batch page.",
       "properties": {
-        "schemaVersions": {
+        "Value": {
```

Comment:
This is a breaking change I believe

Reply:
I can switch it back; probably best to avoid a breaking change if things were working with this naming scheme before